ProcessMaker/processmaker
# FOUR-9362 Log error and continue running if settings are not set

## Issue & Reproduction Steps
A process errors when run and some settings are not set for scripts and data sources

## Solution
Continue running a process if some script or data connector settings are not set. Log the error in the data object

## How to Test

Create a process and drag a script and data connector node onto the canvas. Do not configure the nodes. Connect them and run the process. The process should complete without errors.

## Related Tickets & Packages
- https://processmaker.atlassian.net/browse/FOUR-9362
- Requires https://github.com/ProcessMaker/package-data-sources/pull/300

## Code Review Checklist

ci:package-data-sources:feature/FOUR-9362
